chart/k8gb/README.md

```diff
@@ -101,8 +101,15 @@ For Kubernetes `< 1.19` use this chart and k8gb in version `0.8.8` or lower.
 | rfc2136.enabled | bool | `false` |  |
 | rfc2136.rfc2136Opts[0].host | string | `"host.k3d.internal"` |  |
 | rfc2136.rfc2136Opts[1].port | int | `1053` |  |
-| rfc2136.rfc2136Opts[2].tsig-secret-alg | string | `"hmac-sha256"` |  |
-| rfc2136.rfc2136Opts[3].tsig-keyname | string | `"externaldns-key"` |  |
+| rfc2136.rfc2136auth.insecure.enabled | bool | `false` | Set to True if insecure updates to the DNS provided can be executed by ExternalDNS |
```

Comment:
@v-esteves there is breaking configuration api change. I would double check if it might break the tests

Reply:
Thanks for the tip @ytsarev . Working on it.